source/_posts/2020-09-18-mac-companion.markdown

```diff
@@ -0,0 +1,67 @@
+---
+title: "One more thing…"
+description: "Introducing Home Assistant Companion for macOS."
+date: 2020-09-18 08:00:00
+date_formatted: "September 18, 2020"
+comments: true
+author: Tom Brien
+author_twitter: tbrien88
+categories: Announcements
+og_image: /images/blog/2020-09-18-mac-companion/social.png
+---
+
+TODO: ADD VIDEO
+
+Wow, what a birthday week it has been! We've had a [new supervisor release](/blog/2020/09/16/supervisor-joins-the-party), one of the largest and most user-driven core [releases](/blog/2020/09/17/release-115), thanks to the [month of What The Heck?!](/blog/2020/08/18/the-month-of-what-the-heck/). We even got [RFID tags](/blog/2020/09/15/home-assistant-tags) right into the heart of Home Assistant, but we're not done yet! We have "One more thing…"
+
+## Introducing Home Assistant Companion for macOS
+
+Like many recent updates to the iOS app, we have [@zacwest](https://github.com/zacwest) to thank for this. Zac has ported the iOS app over to Mac and added some great new features specifically for the Mac. If you're a Windows user, don't worry, you can integrate your PC with the great [IOT Link](https://iotlink.gitlab.io/index.html) tool.
```

Comment:
Maybe let's put the Windows reference on a new line? 
```suggestion
Like many recent updates to the iOS app, we have [@zacwest](https://github.com/zacwest) to thank for this. Zac has ported the iOS app over to Mac and added some great new features specifically for the Mac.

_If you're a Windows user, don't worry, you can integrate your PC with the great [IOT Link](https://iotlink.gitlab.io/index.html) tool._
```

Reply:
Makes sense :)